### ISSUE TITLE
Addition to building on Windows NT

### ISSUE THREAD

**Issue by Kawaxte** [up209d/ResourcesSaverExt#64]
Since I'm a simpleton, I use Windows like every person with no regards for `self.privacy`. I needed this extension but didn't want to set up a Linux VM because of it, so here's what I used to build it on Windows 10:
```json
  "scripts": {
    "dev": "yarn cp && parcel watch ./src/*.html --no-hmr --dist-dir unpacked2x",
    "dev-serve": "yarn cp && parcel ./src/*.html --port 20987 --dist-dir unpacked2x",
    "build": "yarn cp && parcel build ./src/*.html --dist-dir unpacked2x",
    "cp": "yarn clean && if not exist unpacked2x mkdir unpacked2x && if exist ./src/static/* xcopy /E /I /H ./src/static/* unpacked2x",
    "clean": "if exist .parcel-cache rmdir /S /Q .parcel-cache && if exist dist rmdir /S /Q dist && if exist unpacked2x rmdir /S /Q unpacked2x",
    "reset": "yarn clean && if exist ./.cache rmdir /S /Q ./.cache && if exist ./.parcel-cache rmdir /S /Q ./.parcel-cache",
    "local": "if exist ./node_modules/parcel-plugin-local rmdir /S /Q ./node_modules/parcel-plugin-local && yarn install --check-files && yarn reset",
    "auto": "if exist output rmdir /S /Q output && babel-node auto.js --presets @babel/preset-env",
    "prettier": "yarn prettier"
  },
```

Maybe you could detect if the user is running on Windows or Linux/macOS and run the according commands?

**Comment by up209d** [comment]
> Since I'm a simpleton, I use Windows like every person with no regards for `self.privacy`. I needed this extension but didn't want to set up a Linux VM because of it, so here's what I used to build it on Windows 10:
> 
> ```json
>   "scripts": {
>     "dev": "yarn cp && parcel watch ./src/*.html --no-hmr --dist-dir unpacked2x",
>     "dev-serve": "yarn cp && parcel ./src/*.html --port 20987 --dist-dir unpacked2x",
>     "build": "yarn cp && parcel build ./src/*.html --dist-dir unpacked2x",
>     "cp": "yarn clean && if not exist unpacked2x mkdir unpacked2x && if exist ./src/static/* xcopy /E /I /H ./src/static/* unpacked2x",
>     "clean": "if exist .parcel-cache rmdir /S /Q .parcel-cache && if exist dist rmdir /S /Q dist && if exist unpacked2x rmdir /S /Q unpacked2x",
>     "reset": "yarn clean && if exist ./.cache rmdir /S /Q ./.cache && if exist ./.parcel-cache rmdir /S /Q ./.parcel-cache",
>     "local": "if exist ./node_modules/parcel-plugin-local rmdir /S /Q ./node_modules/parcel-plugin-local && yarn install --check-files && yarn reset",
>     "auto": "if exist output rmdir /S /Q output && babel-node auto.js --presets @babel/preset-env",
>     "prettier": "yarn prettier"
>   },
> ```
> 
> Maybe you could detect if the user is running on Windows or Linux/macOS and run the according commands?

Good idea, however you can always use a shell emulator like minTTY or xTerm to run unix compatible commands right

**Comment by Kawaxte** [comment]
> > Since I'm a simpleton, I use Windows like every person with no regards for `self.privacy`. I needed this extension but didn't want to set up a Linux VM because of it, so here's what I used to build it on Windows 10:
> > ```json
> >   "scripts": {
> >     "dev": "yarn cp && parcel watch ./src/*.html --no-hmr --dist-dir unpacked2x",
> >     "dev-serve": "yarn cp && parcel ./src/*.html --port 20987 --dist-dir unpacked2x",
> >     "build": "yarn cp && parcel build ./src/*.html --dist-dir unpacked2x",
> >     "cp": "yarn clean && if not exist unpacked2x mkdir unpacked2x && if exist ./src/static/* xcopy /E /I /H ./src/static/* unpacked2x",
> >     "clean": "if exist .parcel-cache rmdir /S /Q .parcel-cache && if exist dist rmdir /S /Q dist && if exist unpacked2x rmdir /S /Q unpacked2x",
> >     "reset": "yarn clean && if exist ./.cache rmdir /S /Q ./.cache && if exist ./.parcel-cache rmdir /S /Q ./.parcel-cache",
> >     "local": "if exist ./node_modules/parcel-plugin-local rmdir /S /Q ./node_modules/parcel-plugin-local && yarn install --check-files && yarn reset",
> >     "auto": "if exist output rmdir /S /Q output && babel-node auto.js --presets @babel/preset-env",
> >     "prettier": "yarn prettier"
> >   },
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Maybe you could detect if the user is running on Windows or Linux/macOS and run the according commands?
> 
> Good idea, however you can always use a shell emulator like minTTY or xTerm to run unix compatible commands right

It would be easier to just natively compile rather than download extra unnecessities.
